qadence/utils.py

```diff
@@ -259,3 +263,31 @@ def validate_values_and_state(
         else:
             if not is_qadence_shape(state, n_qubits) or state.shape[0] > 1:
                 raise ValueError("Jax only supports unbatched states.")
+
+
+def one_qubit_projector(state: str, target: int) -> Projector:
```

Comment:
Yeah, what I was suggesting is to create some partially applied functions and constants like:

```python
P0 = partial(one_qubit_projector, "0")
P0_mat= one_qubit_projector_matrix("0")
```

To be then used elsewhere directly.

```python
P0
```

Reply:
Thank you for the clarification. I added the suggested code. (Due to the circular import, I moved some import statements in some files.)